graphql_compiler/compiler/validation.py

```diff
@@ -55,8 +55,8 @@ def validate_schema_and_query_ast(schema, query_ast):
         ]
     )
 
-    # This directive is supported and ignored by the compiler, since it is meant as an indication
-    # to the user that a field should not be used.
+    # The following directives are supported and ignored by the compiler,
+    # since they are meant to communicate user-facing information.
     supported_default_directive = frozenset(
```

Comment:
nit: Missing an `s` after `supported_default_directive`. 

This is arguably an existing error so I'll approve the PR as is. If you want to fix it though in this PR feel free. I'll approve the PR immediately.

Reply:
Good catch, thanks!